toolbox/db/db_add.m

```diff
@@ -162,19 +162,19 @@
 %                 delfile = bst_fullfile(ProtocolInfo.SUBJECTS, sSubject.Anatomy(1).FileName);
 %             end
         case 'channel'
-            sChannel = sql_query(sqlConn, 'select', 'FunctionalFile', 'FileName', struct('Study', iTarget, 'Type', 'channel'));
-            if ~isempty(sChannel)
-                delfile = bst_fullfile(ProtocolInfo.STUDIES, sChannel.FileName);
+            sFile = db_get('FunctionalFile', sqlConn, struct('Study', iTarget, 'Type', 'channel'));

```

Comment:
This is not equivalent to the previous query. It gets all fields rather than only the 'FileName' field. Same comment for all modifications in this file.

Reply:
Solved in 84736dc